CMakeLists.txt

```diff
@@ -931,16 +932,15 @@ endif()
 set(QGIS_INSTALL_SYS_LIBS TRUE CACHE BOOL "If set to TRUE install all required system libs in the output package")
 
 #############################################################
-# Python build dependency
+# Python
 
-find_package(PythonLibrary REQUIRED)
 set(MIN_PYTHON_VERSION "3.7")
-if(${PYTHON_SHORT_VERSION} VERSION_LESS ${MIN_PYTHON_VERSION})
-    message(FATAL_ERROR "Python version ${PYTHON_SHORT_VERSION} is too old. Minimum Python version is ${MIN_PYTHON_VERSION}.")
-endif()
+set(Python_FIND_FRAMEWORK "LAST")
 
-#############################################################
-# Python bindings
+find_package(Python ${MIN_PYTHON_VERSION} REQUIRED COMPONENTS Interpreter Development)
+message("-- Found Python executable: ${Python_EXECUTABLE}")
+message("-- Python library: ${Python_LIBRARIES}")
+message("-- Python site-packages: ${Python_SITEARCH}")
```

Comment:
Can you report the python version here too?

Reply:
```
  -- Found Python: /usr/bin/python3.8 (found suitable version "3.8.5", minimum required is "3.7") found components: Interpreter Development 
```

Reply:
yes, but only on first run if I am right. Later on, you don't see this anymore. I completed the line.